content/livesync/database-connector.textile

```diff
@@ -150,11 +282,11 @@ event: message
 data: {"id":"1","connectionId":"CaqkrZ2N_0","timestamp":1700069266050,"encoding":"json","extras":{"headers":{"x-ably-models-event-uuid":"1"}},"channel":"foo","data":"{}","name":"test"}
 ```
 
-h2(#new-db). Run a new database
+h3(#new-database). Run with a new database
 
 Use "Railway":https://railway.app/ with a PostgreSQL plugin instead of using your "existing database":#existing-db.
```

Comment:
I think it's important to specify what is Railway and why we are asking to use it when they choose a new db option.